american-journal-of-medical-genetics.csl

```diff
@@ -24,13 +24,13 @@
   </info>
   <macro name="editor">
     <names variable="editor" delimiter=", ">
-      <name name-as-sort-order="all" sort-separator=", " initialize-with="" delimiter=", " delimiter-precedes-last="always"/>
+      <name name-as-sort-order="all" sort-separator=", " initialize-with="" delimiter=" " delimiter-precedes-last="always"/>
```

Comment:
@jxchong, this isn't quite right. The "delimiter" attribute on `<name/>` affects the delimiter between subsequent names. To change the delimiter between "Lastname Initials", you have to change the "sort-separator" attribute value.

That said, which instructions are you looking at, exactly? At http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291552-4833/homepage/ForAuthors.html (the "documentation" link of this style, for American Journal of Medical Genetics Part A), different examples are given:

> For Journals: Thomas, G. H. (1996). High male:female ratio of germ-line mutations: An alternative explanation for postulated gestational lethality in males in X-linked dominant disorders. American Journal of Human Genetics, 58, 1364–1368.
>
>Books:Cassidy SB, Allanson JE, editors. 2010. Management of Genetic Syndromes, Third Edition, Hoboken, NJ: John Wiley and Sons.
>
>Chapter in Book:Battaglia A. 2010. Wolf-Hirschhorn syndrome. In: Cassidy SB, Allanson JE, editors, Management of Genetic Syndromes, Third Edition, Hoboken, NJ: John Wiley and Sons, pp 249-262.

and a recent article from the journal, http://onlinelibrary.wiley.com/doi/10.1002/ajmg.a.38220/pdf, uses comma-space in names, at least for journal articles: "Tluczek, A., Orland, K. M., & Cavanagh, L". The same article also uses parentheses around in-text citations, instead of brackets.